web/yo/app/views/drugs.html

```diff
@@ -0,0 +1,39 @@
+<div class="drugs">
+        <h3 class="headerStyle">Curated Drugs</h3>
+        <table class="table" datatable="ng" dt-options="dtOptions" dt-columns="dtColumns">
+            <thead>
+            <tr class="dashedHeader">
+                <th>Name</th>
+                <th>NCI Thesaurus Code</th>
+                <th>Description</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr data-ng-repeat="drug in drugList">
+                <td ng-include="getTemplate(drug)">
+                </td>
+                <td>{{ drug.NCI }}</td>
+                <td>{{ drug.description }}</td>
+            </tr>
+            </tbody>
+        </table>
+
+        <div class="col-sm-9">
+            <input type="text" class="form-control" ng-model="suggestedDrug" uib-typeahead="drug as drug for drug in processSearchDrugs($viewValue)" typeahead-min-length="1"/>
+        </div>
+        <div class="col-sm-3">
+            <button class="btn btn-success" style="margin-left: 20px" ng-click="addDrug(suggestedDrug)">Add Drug</button>
+        </div>
+
+        <script type="text/ng-template" id="display">
+            <button type="button" class="btn btn-warning buttonStyle " uib-tooltip-placement="top" uib-tooltip="Modify Name" ng-click="editDrugName(drug)"><i class="fa fa-edit"></i>
```

Comment:
Why don't you just change the status here? The function is not needed

Reply:
Now, it's changeDrugName($index). Using $index to make sure that there is only one line changed to edit mode when clicks the changeDrugName button.